src/Compilers/Core/Portable/Operations/DeclarationKind.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    public enum DeclarationKind
```

Comment:
>DeclarationKind [](start = 16, length = 15)

VariableDeclarationKind?

Reply:
I also like this suggestion, it follows the rest of our Kind enums.

---
In reply to: [333643366](https://github.com/dotnet/roslyn/pull/39125#discussion_r333643366) [](ancestors = 333643366)